tests/test_model_testing.py

```diff
@@ -387,3 +388,33 @@ async def test_test_does_not_use_rules(tmp_path: Path, default_agent: Agent):
     test_trackers = generator.generate_story_trackers()
     assert len(test_trackers) == 1
     assert test_trackers[0].sender_id == test_name
+
+
+async def test_nlu_crossvalidation_uses_autoconfig(tmp_path):
+    """Tests that `perform_nlu_cross_validaton` doesn't raise InvalidConfigException."""
+    from rasa.model_testing import perform_nlu_cross_validation
+    from multiprocessing import Process
+
+    config = """language: en
+pipeline:
+policies:
+    """
+    config_file = tmp_path / "empty_config.yml"
+    config_file.write_text(config)
+    training_data_file = "data/test_moodbot/data/nlu.yml"
+    domain_file = "data/test_moodbot/domain.yml"
+
+    test_data_importer = TrainingDataImporter.load_from_dict(
+        training_data_paths=[training_data_file], domain_path=domain_file,
+    )
+    nlu_data = await test_data_importer.get_nlu_data()
+
+    # start cross validation in an outside process to monitor if it fails
```

Comment:
What happens if you call it in the same process?

Reply:
Well, the thing is we want to see if it trains using autoconfig. However, autoconfig has 100 epochs as a default value. That means it will run really long. To not waste time, I stop the process after the first couple seconds. Before this fix, by then a InvalidConfigException would have been raised. Does this make sense?

Reply:
The test is a bit different now, as I am calling rasa test directly, but the concept is the same